src/main/resources/view/HelpWindow.css

```diff
@@ -17,3 +17,65 @@
 #helpMessageContainer {
     -fx-background-color: derive(#1d1d1d, 20%);
 }
+
+#commandTable {
+    -fx-background-color: derive(#1d1d1d, 20%);
+    -fx-text-fill: white;
+    -fx-border-color: gray;
+    -fx-border-width: 1px;
+}
+
+#commandTable .column-header {
+    -fx-background-color: transparent;
+    -fx-border-color: gray;
+    -fx-border-width: 0 1px 1px 0;
+}
+
+#commandTable .column-header .label {
+    -fx-text-fill: white;
+    -fx-font-weight: bold;
+    -fx-padding: 5px;
+}
+
+#commandTable .column-header-background {
+    -fx-background-color: derive(#1d1d1d, 10%);
+}
+
+#commandTable .table-cell {
+    -fx-text-fill: white;
+    -fx-border-color: gray;
+    -fx-border-width: 0 1px 1px 0;
+    -fx-padding: 5px;
+}
+
+#commandTable .table-row-cell {
+    -fx-background-color: derive(#1d1d1d, 20%);
+}
+
+#commandTable .table-row-cell:odd {
```

Comment:
Is there a reason for these `:odd` and `:even` cells? It looks like they are identical, and the help window opens as expected even when I comment out both the odd and even cells. Might be compressed to minimise code duplication

Reply:
ah, it was originally used to alternate shade of the colour for each row. good catch!